rush-plugins/rush-bridge-cache-plugin/README.md

```diff
@@ -0,0 +1,47 @@
+# @rushstack/rush-bridge-cache-plugin
+
+This is a Rush plugin that lets you to add an optional `--set-cache-only` flag to Rush's phased commands to bypass the actual _action_ of the script (build, test, lint - whatever you have configured), and just populate the cache from the action as though the action had already been performed by Rush.
```

Comment:
I personally prefer the model of the command line parameter name being specified in the plugin's config file and being forced to match a command line parameter defined in command-line.json. It's slightly more work for repo maintainers to onboard but guarantees there will not be collisions in parameter naming.